main.tf

```diff
@@ -21,8 +22,7 @@ resource "aws_security_group" "runner" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    #cidr_blocks = ["${var.bastion_ip}/32"] # only from bastion
-    cidr_blocks = ["0.0.0.0/0"] # only from bastion
+    cidr_blocks = ["${var.bastion_ip}/32"]
```

Comment:
Looks like `bastion_ip` becomes a mandatory parameter for module. But what if I don't have or don't need bastion instance at all?

Reply:
```suggestion
    self        = "true"
```

Reply:
Removed var.bastion_ip variable. By default access available only from SG members